doc/source/usage/tutorial/Parallelization/Direct_chunk_read.ipynb

```diff
@@ -0,0 +1,1126 @@
+    "fig,ax = subplots()\n",
+    "ax.imshow(frame)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "17e3d7e2-de22-4d40-8230-39295f04e239",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Performances of the different algorithms for azimuthal integration of Eiger 4M image\n",
+      "Using algorithm histogram : 427 ms ± 11.7 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)\n",
+      "Using algorithm csc       : 39.8 ms ± 2.64 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)\n",
+      "Using algorithm csr       : 60.2 ms ± 10.6 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Performances of the different algorithms for azimuthal integration of Eiger 4M image\")\n",
+    "for algo in (\"histogram\", \"csc\", \"csr\"):\n",
+    "    print(f\"Using algorithm {algo:10s}:\", end=\" \")\n",
+    "    %timeit ai.integrate1d(img, nbins, method=(\"full\", algo, \"cython\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "22969c56-1d10-4950-aebe-699da628858e",
+   "metadata": {},
+   "source": [
+    "**Note:** The full pixel splitting is time consuming and handicaps the histogram algorithm while both sparse-matrix methods are much faster since they cache this calculation in the sparse matrix.\n",
+    "\n",
+    "The compared performances of sparse-matrix methods is rather surprizing since the CSC algorithm, single threaded, is faster than the CSR which runs in parallel over 2x32 cores.\n",
+    "This result is the combination of two facotors:\n",
+    "1. The computer is built with two processors/sockets controling each its own memory. We call this a **Non Uniform Memory Access** computer and can be checked with `numactrl --hardware`. The CSR matrix multiplication will dispatch work on both processors and thus, needs to transfer part of the image from one NUMA subsystem to the other, which is slow (32/10 x slower compared to a single-socket access). \n",
+    "2. The very large cache of this processor: 512MB are reported by `lscpu`, but a more precise tool, `lstopo` describes them as 32MB of L3 cache shared between 4 cores. This very large cache allows the complete frame and the sparse matrix to be pre-fetched which is a great advantage for the CSC algorithm.\n",
+    "\n",
+    "Running the very same benchmark on an Intel 2-socket server would remove the point 2, while running on a singe socket intel workstation would remove both points and the normal results would be that CSR should be faster than CSC. The best performances on can get with the CSR algorithm should be obtained when using 4 cores (sharing the same cache L3) out of 64 on this computer. This can be done by setting the environment variable **OMP_NUM_THREADS**. Unfortunately, it also requires to restart the process, thus cannot be demonstrated easily in the notebook (without restarting). \n",
```

Comment:
```suggestion
    "Running the very same benchmark on an Intel 2-socket server would remove the point 2, while running on a singe socket intel workstation would remove both points and the normal results would be that CSR should be faster than CSC. The best performances one can get with the CSR algorithm should be obtained when using 4 cores (sharing the same cache L3) out of 64 on this computer. This can be done by setting the environment variable **OMP_NUM_THREADS** before importing `h5py` and `hdf5plugin`. Unfortunately, it requires to restart the process, thus cannot be demonstrated easily in the notebook (without restarting). \n",
```